luigi/tools/parse_task.py

```
@@ -55,7 +58,10 @@ def id_to_name_and_params(task_id):
         pp.ZeroOrMore(parameter) +
         pp.Literal(')').suppress())
 
-    parsed = parser.parseString(task_id).asDict()
+    try:
+        parsed = parser.parseString(task_id).asDict()
+    except Exception as e:
+        logger.error("Exception while parsing task_id: " + e)
```

I think we want to re-raise the exception after logging.  I think this is going to fail on line 65 where parsed is undefined.

---

Ah, yeah I missed that the log message doesn't have the actual task id in it. :)